app/assess/helpers.py

```diff
@@ -44,6 +44,8 @@ def set_application_status_in_overview(application_overviews):
             "flag_type"
         ] in ["FLAGGED", "STOPPED"]:
             status = "QA_COMPLETED"
+        elif len(overview["flags"]) > 1:
```

Comment:
I've called it `flags_v2` in the new version so as to leave the old stuff there for now.,